test/functional/rpc_misc.py

```diff
@@ -57,7 +57,7 @@ def run_test(self):
         self.log.info("test logging rpc and help")
 
         # Test logging RPC returns the expected number of logging categories.
-        assert_equal(len(node.logging()), 27)
+        assert 26 <= len(node.logging()) <= 27
```

Comment:
What is the point of this other than causing merge conflicts? Might be best to remove.

Reply:
Considered removing, done